array-manipulation-2/src/take-right.js

```diff
@@ -0,0 +1,16 @@
+/* exported takeRight */
+// Make new storage for the output
+// check the array one by one starting from  the array length minus the count given
+// check until the end
+// If the count is undefined or greater than the length return the values of the array as output
+// Otherwise store the values checked from the starting point to the end
+// return the storage as output
+function takeRight(array, count) {
+  var newArr = [];
+  for (var i = array.length - count; i < array.length; i++) {
+    if (count > array.length || array === 'undefined') {
+      return array;
```

Comment:
What is the purpose of `array === "undefined"`? 😕

Reply:
Also, you should do this conditional above the loop, you don't need to do this check on every iteration of the loop.